tck/tck-dist/src/main/assembly/assembly.xml

```diff
@@ -34,7 +34,40 @@
          <source>artifact-install.sh</source>
          <outputDirectory>artifacts</outputDirectory>
       </file>
+      <file>
+         <source>../tck-docs/userguide/target/generated-docs/Jakarta-JSON-Processing-TCK-Users-Guide.pdf</source>
```

Comment:
This will work as long as folder structure and/or filenames remain untouched.

Better solution would be if the `userguide` artifact builds a zip with pdf/html files and the dist artifact should take that, unzip it to its own target folder and take it from there.

Reply:
currently implementation has dependency on folder structure, sure we can enhance the assembly to unzip from doc artifacts in future.